package.json

```diff
@@ -6,13 +6,16 @@
   "scripts": {
     "lint:fix": "eslint -c .eslintrc --fix lib/*.js lib/*/*.js",
     "lint": "eslint -c .eslintrc lib/*.js lib/**/*.js test/**/*.js",
-    "test:unit": "nyc mocha test/unit/*.test.js test/unit/**/*.test.js",
+    "test:unit": "mocha test/unit/*.test.js test/unit/**/*.test.js",
     "test:unit:debug": "mocha --inspect --debug-brk test/unit/*.test.js test/unit/**/*.test.js",
+    "test:integration": "mocha test/integration/*.test.js",
+    "test:integration:debug": "mocha --inspect --debug-brk test/integration/*.test.js",
     "pretest": "npm run lint:fix",
-    "test": "npm run test:unit",
+    "test": "nyc mocha test/unit/*.test.js test/unit/**/*.test.js test/integration/*.test.js",
```

Comment:
You can do this so you don't have to redefine those file filters
```
"test:all": "npm run test:unit && npm run test:integration",
"test": "nyc npm run test:all",
```

Reply:
Will do in a later PR